output/mapping/F14_2014.csv

```diff
@@ -44,31 +44,31 @@ xpath,label-key,index,comment,guidance
 /CHANGES/CHANGE/WHERE,,,,""
-/CHANGES/CHANGE/WHERE/SECTION,section_no,,,""
+/CHANGES/CHANGE/WHERE/SECTION,section_no,,,Map to `.where.section`.
 /CHANGES/CHANGE/OLD_VALUE,icar_instead,,,""
 /CHANGES/CHANGE/NEW_VALUE,icar_read,,,""
-/CHANGES/CHANGE/WHERE/LOT_NO,lot_number,,,""
-/CHANGES/CHANGE/WHERE/LABEL,icar_text_corr_place,,,""
-/CHANGES/CHANGE/OLD_VALUE/NOTHING,,,,""
-/CHANGES/CHANGE/NEW_VALUE/NOTHING,,,,""
-/CHANGES/CHANGE/OLD_VALUE/TEXT,,,,""
-/CHANGES/CHANGE/NEW_VALUE/TEXT,,,,""
+/CHANGES/CHANGE/WHERE/LOT_NO,lot_number,,,Map to `.where.lot`.
+/CHANGES/CHANGE/WHERE/LABEL,icar_text_corr_place,,,Map to `.where.label`.
+/CHANGES/CHANGE/OLD_VALUE/NOTHING,,,,Set `.oldValue.text` to `null`.
+/CHANGES/CHANGE/NEW_VALUE/NOTHING,,,,Set `.newValue.text` to `null`.
+/CHANGES/CHANGE/OLD_VALUE/TEXT,,,,Map to `.oldValue.text`.
+/CHANGES/CHANGE/NEW_VALUE/TEXT,,,,Map to `.newValue.text`.
 /CHANGES/CHANGE/OLD_VALUE/CPV_MAIN,,,,""
 /CHANGES/CHANGE/NEW_VALUE/CPV_MAIN,,,,""
-/CHANGES/CHANGE/OLD_VALUE/CPV_MAIN/CPV_CODE,cpv_main,,,""
-/CHANGES/CHANGE/NEW_VALUE/CPV_MAIN/CPV_CODE,cpv_main,,,""
-/CHANGES/CHANGE/OLD_VALUE/CPV_MAIN/CPV_SUPPLEMENTARY_CODE,cpv_supplem,,,""
-/CHANGES/CHANGE/NEW_VALUE/CPV_MAIN/CPV_SUPPLEMENTARY_CODE,cpv_supplem,,,""
+/CHANGES/CHANGE/OLD_VALUE/CPV_MAIN/CPV_CODE,cpv_main,,,Map to `.oldValue.mainCpv`.
+/CHANGES/CHANGE/NEW_VALUE/CPV_MAIN/CPV_CODE,cpv_main,,,Map to `.newValue.mainCpv`
+/CHANGES/CHANGE/OLD_VALUE/CPV_MAIN/CPV_SUPPLEMENTARY_CODE,cpv_supplem,,,Map to `.oldValue.mainCpvSupplementary`
+/CHANGES/CHANGE/NEW_VALUE/CPV_MAIN/CPV_SUPPLEMENTARY_CODE,cpv_supplem,,,Map to `.newValue.mainCpvSupplementary`
 /CHANGES/CHANGE/OLD_VALUE/CPV_ADDITIONAL,,,,""
-/CHANGES/CHANGE/OLD_VALUE/CPV_ADDITIONAL/CPV_CODE,,,,""
-/CHANGES/CHANGE/OLD_VALUE/CPV_ADDITIONAL/CPV_SUPPLEMENTARY_CODE,,,,""
+/CHANGES/CHANGE/OLD_VALUE/CPV_ADDITIONAL/CPV_CODE,,,,Map to `.oldValue.additionalCpv`
+/CHANGES/CHANGE/OLD_VALUE/CPV_ADDITIONAL/CPV_SUPPLEMENTARY_CODE,,,,Map to `.oldValue.additionalCpvSupplementary`
 /CHANGES/CHANGE/NEW_VALUE/CPV_ADDITIONAL,,,,""
-/CHANGES/CHANGE/NEW_VALUE/CPV_ADDITIONAL/CPV_CODE,,,,""
-/CHANGES/CHANGE/NEW_VALUE/CPV_ADDITIONAL/CPV_SUPPLEMENTARY_CODE,,,,""
-/CHANGES/CHANGE/OLD_VALUE/DATE,date,,,""
-/CHANGES/CHANGE/NEW_VALUE/DATE,date,,,""
-/CHANGES/CHANGE/OLD_VALUE/TIME,time,,,""
-/CHANGES/CHANGE/NEW_VALUE/TIME,time,,,""
+/CHANGES/CHANGE/NEW_VALUE/CPV_ADDITIONAL/CPV_CODE,,,,Map to `.newValue.additionalCpv`
+/CHANGES/CHANGE/NEW_VALUE/CPV_ADDITIONAL/CPV_SUPPLEMENTARY_CODE,,,,Map to `.newValue.additionalCpvSupplementary`
+/CHANGES/CHANGE/OLD_VALUE/DATE,date,,,Map to `.oldValue.date`
+/CHANGES/CHANGE/NEW_VALUE/DATE,date,,,Map to `.newValue.date`
+/CHANGES/CHANGE/OLD_VALUE/TIME,time,,,Map to `.oldValue.time`
+/CHANGES/CHANGE/NEW_VALUE/TIME,time,,,Map to `.newValue.time`
```

Comment:
The XML validation is strict enough according to the XSD that we can format this as a single date-time field in JSON. We can use the same language as other datetime mappings.